benchmarks/triton_kernels_benchmark/gemm_tensor_of_ptr_benchmark.py

```diff
@@ -161,198 +120,24 @@ def matmul_kernel_batched(
-    elif provider == 'triton':
-        assert len(a.shape) == len(b.shape), 'Incompatible sizes'
-        if len(a.shape) == 3:
-            c = torch.zeros((B, M, N), device='xpu', dtype=torch.float32)
-        else:
-            assert len(a.shape) == 2, 'Expecting shape of length 2'
-            c = torch.zeros((M, N), device='xpu', dtype=torch.float32)
-        triton_fn = lambda: matmul(a, b, c, transpose_a=TRANSPOSE_A, transpose_b=TRANSPOSE_B)
-        torch_fn = lambda: torch.matmul(torch_a, torch_b).to(torch.float32)
-        rtol = 1e-2 if a.dtype == torch.bfloat16 else 1e-3
-        benchmark_suit.assert_close(triton_fn, torch_fn, atol=1e-4, rtol=rtol, err_msg='triton to torch')
-        _, min_ms, max_ms, mean_ms, cv = benchmark_suit.do_bench(triton_fn, n_warmup=10, n_repeat=10,
-                                                                 quantiles=quantiles)
-    elif provider == 'xetla':
-        if B == 1:
-            c = torch.zeros((M, N), device='xpu', dtype=torch.float32)
-            cnt = torch.zeros((M, N), device='xpu', dtype=torch.int32)
-        else:
-            c = torch.zeros((B, M, N), device='xpu', dtype=torch.float32)
-            cnt = torch.zeros((B, M, N), device='xpu', dtype=torch.int32)
-        name = f'gemm_shape_{B}_{M}_{K}_{N}'
-        # FIXME: Use gemm_streamk_benchmark.py when Triton streamk can get
-        # better performance.
-        if (B, M, N, K) == (1, 3072, 3072, 4096):
-            name = 'gemm_streamk_shape_3072_4096_3072'
-        func = getattr(xetla_kernel, name)
-
-        def xetla_func_with_acc_allocation():
-            # allocating `acc` matrix on every function call, to be as similar as
-            # possible to the triton kernel, which also does this on every call.
-            if B == 1:
-                acc = torch.zeros((M, N), device='xpu', dtype=torch.float32)
-            else:
-                acc = torch.zeros((B, M, N), device='xpu', dtype=torch.float32)
-            return func(a, b, c, acc, cnt)
-
-        xetla_fn = xetla_func_with_acc_allocation
-        torch_fn = lambda: torch.matmul(a, b).to(torch.float32)
-
-        # benchmark_suit.assert_close(xetla_fn, torch_fn, atol=1e-4, rtol=1.0, err_msg='xetla to torch')
-        _, min_ms, max_ms, mean_ms, cv = benchmark_suit.do_bench(xetla_fn, n_warmup=10, n_repeat=10,
-                                                                 quantiles=quantiles)
-    else:
-        raise NotImplementedError(f'Unsupported provider {provider}')
-
-    tflops = lambda ms: 2 * B * M * N * K * (1e-12) / (ms * 1e-3)
-    gbps = lambda ms: B * (2 * (M * K + K * N) + 4.0 * (M * N)) * (1e-9) / (ms * 1e-3)
-
-    return (gbps(mean_ms), gbps(max_ms), gbps(min_ms)), (tflops(mean_ms), tflops(max_ms), tflops(min_ms)), cv
+        transpose_a=transpose_a,
+        transpose_b=transpose_b,
+    )
 
 
 if __name__ == '__main__':
-    benchmark.run(show_plots=False, print_data=True)
+    _benchmark = gemm_benchmark.get_benchmark(
```

Comment:
I believe that this is the reason for the failure of CI.
```suggestion
    _benchmark = get_benchmark(
```